app/src/lib/stores/app-store.ts

```diff
@@ -2373,6 +2373,27 @@ export class AppStore extends TypedBaseStore<IAppState> {
             }
           }
         }
+
+        if (
+          gitStore.tip.kind === TipState.Valid &&
+          gitStore.currentRemote !== null
+        ) {
```

Comment:
Given that this section doesn't do anything but write to the log at this time could we add a feature flag to ensure it's isolated from production while we work on this?

Reply:
Addressed in dc8a86a12e42e4e90ded7054b4ef15d6ef372c01